blog/_posts/2019-09-05-put-down-particle-accelerator.md

```diff
@@ -0,0 +1,78 @@
+---
+slug: put-down-particle-accelerator
+date: 2019-09-05
+author: dan
+layout: blog
+title: "Microservice Devs: Put Down the Particle Accelerator"
+subtitle: "You Need a Chemistry Set"
+image: slac.jpg
```

Comment:
![Screen Shot 2019-09-05 at 3 34 35 PM](https://user-images.githubusercontent.com/6332648/64374874-b2c80900-cff2-11e9-87b5-9d50332a5a54.png)

do you have a higher-res image for this? It looks not-great on the blog homepage.